pkg/auth/worker_interceptor_test.go

```diff
@@ -0,0 +1,27 @@
+package auth
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+func TestWorkerUnary(t *testing.T) {
+	interceptor := &WorkerInterceptor{
+		client: &fakeInternalServerClient{
```

Comment:
fakeclient.AuthorizeWorker returns not implemented error

Reply:
fixed!